pages/linux/lxi.md

```diff
@@ -0,0 +1,24 @@
+# lxi
+
+> Tool for controling LXI compatible instruments such as oscilloscopes.
```

Comment:
```suggestion
> Control `LXI` compatible instruments such as oscilloscopes.
```

Controlling was also spelt wrong

Reply:
Is there some new rule for the \` around LXI change, that I've failed to acknowledge, please?

Reply:
No, but it makes more sense in the fact that it sounds like a standard, if its just a generic software then feel free to just change the other bit suggested

Reply:
I'm not sure the backticks make sense here.
```suggestion
> Control LXI compatible instruments such as oscilloscopes.
```